x-pack/plugins/aiops/public/components/spike_analysis_table/use_copy_to_clipboard_action.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiCopy, EuiToolTip, EuiButtonIcon } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+import { isSignificantTerm, type SignificantTerm } from '@kbn/ml-agg-utils';
+
+import { getTableItemAsKuery } from './get_table_item_as_kuery';
+import type { GroupTableItem, TableItemAction } from './types';
+
+const copyToClipboardSignificantTermMessage = i18n.translate(
+  'xpack.aiops.spikeAnalysisTable.linksMenu.copyToClipboardSignificantTermMessage',
+  {
+    defaultMessage: 'Copy field/value pair as KUERY filter to clipboard',
```

Comment:
Not sure if these messages should use `KUERY` or `Kuery` 🤔

Reply:
Replace `KUERY filter` with `KQL syntax` to be in line with the wording in the Discover search bar placeholder in 2eafaec.